frontend/src/routes/index.svelte

```diff
@@ -169,79 +165,21 @@
-                                        </div>
-                                    {/each}
-                                </div>
-                            {:else}
-                                <div class="-space-x-4 avatar-group">
-                                    {#each media.specific_providers.slice(0, SHOWN_PROVIDERS - 1) as provider}
-                                        <div class="avatar">
-                                            <div class="w-12 h-12">
-                                                <img src="{IMG_URL}{provider.logo_path}"
-                                                    alt="provider logo">
-                                            </div>
-                                        </div>
-                                    {/each}
-                                    <div class="avatar placeholder">
-                                        <div class="w-12 h-12 rounded-full bg-neutral-focus text-neutral-content">
-                                            <span>+{providerAmounts[mediaIndex] - SHOWN_PROVIDERS  + 1}</span>
-                                        </div>
-                                    </div>
-                                </div>
-                            {/if}
-                        </div>
-                    {/each}
-                </div>
-                <div class="flex space-x-1 justify-center p-1">
-                    {#if currentMediaAmount < media.nbHits}
-                        <button
-                            on:click={() => { changeMediaAmount('loadmore') }}
-                            id="loadmore"
-                            type="button"
-                            class="btn">
-                            Show more
-                        </button>
-                    {/if}
-                    {#if currentMediaAmount > MEDIA_START_AMOUNT}
-                        <button
-                            on:click={() => { changeMediaAmount('loadless') }}
-                            id="loadless"
-                            type="button"
-                            class="btn">
-                            Show less
-                        </button>
-                    {/if}
-                </div>
-            {:else}
-                <NoResults
-                    currentProviders={$currentProviders}
-                    currentGenres={$currentGenres}
-                    input={input}
-                />
-            {/if}
-        {/if}
+        <MediaSearch
+            media={media}
+            lowResUrl={LOW_RES_IMG_URL}
+            imgUrl={IMG_URL}
+            providerAmounts={providerAmounts}
+            currentCountry={$currentCountry}
+            shownProviders={SHOWN_PROVIDERS}
+            currentProviders={$currentProviders}
+            bind:currentMediaAmount={currentMediaAmount}
+            mediaStartAmount={MEDIA_START_AMOUNT}
+            showButtonAmount={SHOW_BUTTON_AMOUNT}
```

Comment:
I'm going to be annoying and mark all constants :)

It would be super clean define these in the components instead of passing them